src/cli/index.js

```diff
@@ -286,6 +286,7 @@ export function main({
     return new Promise((resolve, reject) => {
       const connectionOptions = {
         port: +mutexPort || constants.SINGLE_INSTANCE_PORT,
+        host: 'localhost',
```

Comment:
It's interesting that it didn't work before given that https://nodejs.org/api/http.html#http_http_request_options_callback (and by extension http.get) defaults to `localhost`?

Reply:
Yeah I think for the http.get call below this does not matter but for the [server.listen](https://nodejs.org/api/http.html#http_server_listen) call it does.